openbsd/OpenBSDMachine.c

```diff
@@ -92,11 +92,12 @@ Machine* Machine_new(UsersTable* usersTable, uid_t userId) {
    char errbuf[_POSIX2_LINE_MAX];
 
    OpenBSDMachine* this = xCalloc(1, sizeof(OpenBSDMachine));
-   Machine* super = &this->super;
+   Machine* super = (Machine*)&this->super;
 
    Machine_init(super, usersTable, userId);
 
-   OpenBSDProcessList_updateCPUcount(this);
+   // Commented this out temporarily to fix build error incorrectly:
+   //OpenBSDProcessList_updateCPUcount(this); // How do I fix it ???
```

Comment:
There seems to be that some code got lost in 72235d8e098d9d79029dca65122605741e1aafad when @natoscott did some refactoring to separate the Machine code from the Process List stuff.

Have a look at that commit (and those nearby) and look for that routine (should have been in `OpenBSDProcessList.c`). The new location for this should be `OpenBSDMachine.c` following the intention of PR #1234 (cf. #1238, #1239, #1240).

Reply:
@time-killer-games @BenBE yep, that's a spot I've missed - please replace OpenBSDProcessList_updateCPUcount with OpenBSDMachine_updateCPUcount here.  Thanks!